src/main/java/uk/gov/justice/probation/courtcaseservice/service/OffenderService.java

```diff
@@ -119,4 +119,7 @@ private ProbationRecord addConvictionsToProbationRecord(ProbationRecord probatio
         return defaultClient.getConvictionRequirements(crn, convictionId).block();
     }
 
+    public OffenderDetail getOffenderDetail(String crn) {
+        return defaultClient.getOffenderDetailByCrn(crn).block();
```

Comment:
Could return a Mono all the way to the controller here which would keep it fully non-blocking. Doesn't really matter that much in the whole scheme of things so up to you

Reply:
Done